workflows/train/azure-automl/classification/job.py

```diff
@@ -0,0 +1,34 @@
+# description: train an Azure AutoML classification model on credit card data
+
+import os
+
+os.system("pip install --upgrade azureml-train-automl")
+
+from azureml.core import Workspace, Experiment, Dataset
+from azureml.train.automl import AutoMLConfig
+
+ws = Workspace.from_config()
+
+data = "https://automlsamplenotebookdata.blob.core.windows.net/automl-sample-notebook-data/creditcard.csv"
+experiment_name = "azure-automl-creditcard-example"
+compute_name = "cpu-cluster"
+
+dataset = Dataset.Tabular.from_delimited_files(data)
+
+automl_settings = {
```

Comment:
this run on what I assume to be a small CSV is taking 60+ minutes and does not seem to be utilizing the cluster very well - needs some edits to make in as a workflow example, else can be a tutorial

Reply:
@ahyerman this still took 45 minutes with the updates, which is way too long for an example workflow

Reply:
@lostmygithubaccount the problem coming from this sample is a bad recommendation from our recommender. We have a work item to track this. The other short term mitigation to speed this up is to remove that algorithm from the recommendation list.

You can do that via the blocked_models parameter.

AutoMLConfig(blocked_models=["LinearSVM", "SVM", "KNN"])

However this probably is something a bit specific to include in a generic sample run.

Reply:
Another option is to explicitly set the iteration timeout to a few min. This will terminate any long running models after timeout expires.

AutoMLConfig(iteration_timeout_minutes=2)